doc/whats_new.rst

```diff
@@ -301,6 +301,8 @@ Changelog
 
 - Handle different time vectors in topography plots using :func:`mne.viz.plot_evoked_topo` by `Jussi Nurminen`_
 
+- Speed up :func:`_mixed_norm_solver_bcd` (called by :func:`mixed_norm` if the :solver: parameter is set to `bcd`) using :func:`func:scipy.linalg.get_blas_func`.
```

Comment:
No private functions in `whats_new.rst`, mention which major public things it will actually help instead

also no need for `func:scipy` you can just have `scipy.` inside the ticks